### DISCUSSION TITLE
can somebody help me, because i need to send the link to my mentor, but it isnt working

### DISCUSSION THREAD

**Discussion by hftwich** [community/community#46990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i have follow the steps they had explain me, but now i'n stil getting the same problem ass olways, and i need this to work out because i need to do this all the time again and again. 
the settings are public and in main, 
i have the html file
and i have the acces online 
still not working 
![trouble](https://user-images.githubusercontent.com/115465180/218166027-168dc47b-7b3f-4452-80d1-c2f840f105d2.jpg)

**Comment by radar-blip** [comment]
You need to have an index.html in your directory (repository)
So, rename `hanniefa.html` to`index.html`

Also, the links to your images: `<img src="img/1.jpg" alt="nagel" width="75" height="80"/>`
should be like this: `<img src="1.jpg" alt="nagel" width="75" height="80"/>`

**Comment by hftwich** [comment]
Thanks a lot!

Op vr 10 feb. 2023 19:31 schreef David ***@***.***>:

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hftwich ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
